src/App.css

```diff
@@ -1,39 +1,38 @@
-.App {
-  text-align: center;
+.calculator-wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 0.25rem;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.result-section {
+  grid-column: 4 span;
+  background-color: #333;
```

Comment:
- [ ] I think the background colour you used here doesn't match the one on the given design. It should be something like grey instead of black. Please also consider expanding the width of your calculator. 

Your Design |  Given Design |
:----------------:|:-------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159504741-ca66409c-3d5e-47bc-afb4-7f6ca2ba3916.png)| ![a2](https://user-images.githubusercontent.com/39523312/159503901-dfdb0c6c-29c9-48b1-8a96-5d075b7a9990.png)

Reply:
Ok i will correct it.